.github/ROADMAP.md

```diff
@@ -12,11 +12,10 @@ The team will release one minor version upgrade each quarter, or whenever a new
 
 
 2022 Q4:
-- 2D Semantic Segmentation task, the team focuses on DeepLabV3 and opens up contribution
-  for UNet/FCN.
-- 3D Object Detection task, the team focuses on providing point cloud data pipeline.
+- 2D Semantic Segmentation task, the team focuses on DeepLabV3, DeepLabV3+ and opens up contribution for UNet/FCN.
+- 3D Object Detection task, the team focuses on providing point cloud data augmentation pipeline, losses and metrics.
 - 2D Instance Segmentation task, the team focuses on MaskRCNN meta arch built on top of FasterRCNN.
-- Text-to-image task, more specifically built on top of StableDiffusion for image impainting and text-to-video.
+- Text-to-image task, more specifically built on top of StableDiffusion for image impainting, textual inversion and text-to-video.
```

Comment:
how about `image-to-image` with `text` guided. [SRC](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/README.md#image-to-image-text-guided-generation-with-stable-diffusion).

Reply:
Along with SD model, [Imagen](https://github.com/lucidrains/imagen-pytorch) and [Dalle-2](https://github.com/lucidrains/DALLE2-pytorch) should be also mentioned and welcomed.

Reply:
..and PARTI

Reply:
those are not even single-machine serve-able, right?